app/models/google_calendar/google_event.rb

```diff
@@ -0,0 +1,27 @@
+module GoogleCalendar
+  class GoogleEvent
+    def initialize(credentials, user_email)
+      @credentials = credentials
+      @user_email = user_email
+    end
+
+    def create(conference_room_id, raw_event_data = {})
+      event_creator = GoogleCalendar::EventCreator.new(credentials)
+      event_creator.create(conference_room_id, raw_event_data)
+    end
+
+    def delete(event_id)
+      event_remover = GoogleCalendar::EventRemover.new(credentials)
+      event_remover.delete(event_id)
```

Comment:
GoogleCalendar::EventRemover.new(credentials).delete(event_id) you don't need to create variable